InteractionBase/src/vis/CommandPrompt.cpp

```diff
@@ -143,7 +145,7 @@ void CommandPrompt::determineChildren()
 {
 	GenericHandler* h = dynamic_cast<GenericHandler*> (handler());
 
-	if (h)
+	if (h && !hideRequested_)
```

Comment:
Hmm, it's not entirely obvious what problem this solves. Can you give me a step-by-step example of the issue?